core/commands/commit/interactors/get_file_content.py

```diff
@@ -13,6 +13,9 @@ async def get_file_from_service(self, commit, path):
                 owner=self.current_owner, repo=commit.repository
             )
             content = await repository_service.get_source(path, commit.commitid)
+            # Sometimes the content may be of type str, so don't decode it
```

Comment:
super nit: this comment could be reworded to explain some of the cases where the content might be a str. Currently idk how much value the comment is bringing